### PR TITLE
Update mod for Stardew Valley 1.6

### PR DESCRIPTION
**Pull request by taylorjwalker**
Made minimal changes to ensure compatibility with Stardew Valley 1.6.
Does not provide backwards compatibility with versions < 1.6.

### CODE REVIEW

**Comment by taylorjwalker** [comment]
Hey @andyruwruw, love the mod and can't wait to see what you're cooking up in `develop`. In the meantime, I hope this can provide value for you and any other Water Bot users.

File: WaterBot/Framework/WaterBotControler.cs [hunk 1]
```diff
@@ -155,7 +153,6 @@ public void water(Point tile)
                 Game1.player.EndEmoteAnimation();
             }
 
-            Game1.player.FarmerSprite.setOwner(Game1.player);
```
Comment (taylorjwalker) [inline WaterBot/Framework/WaterBotControler.cs:158]: There simply doesn't seem to be an equivalent method for this in 1.6. I have not noticed any adverse effects after removing it.

**Comment by taylorjwalker** [comment]
Here's the zip in case anyone wants to check it out without building it:
[WaterBot 2.0.0.zip](https://github.com/andyruwruw/stardew-valley-water-bot/files/14808184/WaterBot.2.0.0.zip)

File: WaterBot/Framework/WaterBotControler.cs (hunk 1 above)
Comment (taylorjwalker) [reply WaterBot/Framework/WaterBotControler.cs:158]: Somehow I missed it, but `SetOwner` exists... fixed in latest commit.

**Comment by andyruwruw** [comment]
Oh you're an absolute legend.

**Pull request merged**